fastlane/Fastfile

```diff
@@ -1127,10 +1129,16 @@ lane :test_without_building do |options|
 
   inject_buildkite_analytics_environment(xctestrun_path: xctestrun_path) if buildkite_ci?
 
+  # Excluding unit tests from parallel testing
+  parallel_testing_value = options[:name] == "UnitTests" ? false : true
```

Comment:
This could be made, IMHO, clearer by switching the check to enable only for UI tests

```suggestion
  # Only run UI tests in parallel
  parallel_testing_value = options[:name] == "UITests"
```

Also, it would be handy to add a line about the fact that we need to do this despite the test plan already specifying this:

```suggestion
  # Only run UI tests in parallel.
  # At the time of writing with Xcode 14.3, we need to explicitly set this value despite using test plans that configure parallelism.
  parallel_testing_value = options[:name] == "UITests"
```

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/9514/commits/8e5bbaef944f12ec2ea0d8f66d21eba0b932d7f7